pkg/cgroup/cgroup_cpu.go

```diff
@@ -88,6 +88,49 @@ func getCgroupCPUHelper(root string) (CPUUsage, error) {
 	return res, nil
 }
 
+// Helper function for getCgroupCPUPeriodAndQuota. Root is always "/", except in tests.
+func getCgroupCPUPeriodAndQuota(root string) (period int64, quota int64, err error) {
+	path, err := detectControlPath(filepath.Join(root, procPathCGroup), "cpu")
+	if err != nil {
+		return
+	}
+
+	// No CPU controller detected
+	if path == "" {
+		err = errors.New("no cpu controller detected")
+		return
+	}
+
+	mount, ver, err := getCgroupDetails(filepath.Join(root, procPathMountInfo), path, "cpu")
+	if err != nil {
+		return
+	}
+
+	if len(mount) == 2 {
+		cgroupRootV1 := filepath.Join(root, mount[0])
+		cgroupRootV2 := filepath.Join(root, mount[1], path)
+		period, quota, err = detectCPUQuotaInV2(cgroupRootV2)
+		if err != nil {
+			period, quota, err = detectCPUQuotaInV1(cgroupRootV1)
+		}
+		if err != nil {
+			return
+		}
+	} else {
+		switch ver[0] {
```

Comment:
Do we need to check the length of `ver`?

Reply:
This is the same as as cgroup_cpu.go of tidb code.